mne/viz/_3d.py

```diff
@@ -1692,6 +1693,10 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         effect with mayavi backend. Defaults  to 'oct6'.
 
         .. versionadded:: 0.15.0
+    title : str | None
+        Title for the figure. If None, the subject name will be used.
+
+        .. versionadded:: 0.17.0
```

Comment:
Enh: Allow `plot_source_estimates` to take a `title` kwarg as an alternative to auto-setting it